packages/terra-pills/src/private/_Pill.jsx

```diff
@@ -0,0 +1,313 @@
+import React, {
```

Comment:
The Pill component isn't private, consumers are meant to import it, I would remove the `private` folder and underscore prefix. The `private` conventions are intended for internal components that consumers should not import directly and aren't considered part of our API contract with the consumers.

Reply:
changed ae07eb46c1332427df41f2b68be6f9007c75930c